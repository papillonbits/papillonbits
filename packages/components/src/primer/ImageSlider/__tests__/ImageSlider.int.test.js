import { regular } from './ImageSlider.int.story'

describe('<ImageSlider />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
