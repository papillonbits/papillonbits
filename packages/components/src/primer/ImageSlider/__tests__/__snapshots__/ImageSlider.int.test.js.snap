// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ImageSlider /> Render must match regular() 1`] = `
<div>
  <div
    class="slider"
  >
    <canvas
      class="canvas"
      height="400"
      width="3050"
    />
  </div>
</div>
`;
